api/config/cdr_config_preprod.json

```diff
@@ -89,11 +89,11 @@
       "cdrVersionId": 6,
       "name": "All of Us Registered Tier Dataset v5",
       "bigqueryProject": "fc-aou-cdr-preprod",
-      "bigqueryDataset": "R2021Q3R5",
+      "bigqueryDataset": "R2021Q3R8",
       "creationTime": "2021-08-10 00:00:00Z",
       "releaseNumber": 7,
       "numParticipants": 329070,
-      "cdrDbName": "r_2021q3_10",
+      "cdrDbName": "r_2021q3_12",
```

Comment:
Did you bump this by 2 just to match CT? Not sure it matters, just curious.

Reply:
yes